source/opt/ir_context.h

```diff
@@ -1208,6 +1212,25 @@ Instruction* IRContext::GetMemberName(uint32_t struct_type_id, uint32_t index) {
   return nullptr;
 }
 
+void IRContext::CloneNames(uint32_t old_id, uint32_t new_id, uint32_t new_len) {
```

Comment:
nit: all those params can be marked as const.

Reply:
Done.